bazel/repository_locations.bzl

```diff
@@ -864,6 +864,36 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         license = "Apache-2.0",
         license_url = "https://github.com/grpc-ecosystem/proto-converter/blob/{version}/LICENSE",
     ),
+    com_google_protofieldextraction = dict(
+        project_name = "proto-field-extraction",
+        project_desc = "Library that supports the extraction from protobuf binary",
+        project_url = "https://github.com/grpc-ecosystem/proto-field-extraction",
+        version = "12090e01883651ea77da7451af045e02c1539291",
+        sha256 = "875a5a9e5a5a39b9c57162e7750edfd56eaf3090cfb61eaebf945260f6a4f2ee",
+        strip_prefix = "proto-field-extraction-{version}",
+        urls = ["https://github.com/grpc-ecosystem/proto-field-extraction/archive/{version}.zip"],
+        use_category = ["dataplane_ext"],
+        extensions = ["envoy.filters.http.grpc_field_extraction"],
+        release_date = "2023-06-30",
+        cpe = "N/A",
+        license = "Apache-2.0",
+        license_url = "https://github.com/grpc-ecosystem/proto-field-extraction/blob/{version}/LICENSE",
+    ),
+    ocp = dict(
```

Comment:
What is this used for?

Reply:
I introduced this dependency for some handy macros. google doesn't release them in github and I found this repo having them.